core/templates/dev/head/components/share/sharing_collection_links_directive.html

```diff
@@ -0,0 +1,74 @@
+<script type="text/ng-template" id="components/sharingCollectionLinks">
```

Comment:
well, yes. I also see only these two plus registerShareCollectionEvent / registerShareExplorationEvent.
I think i'll create a separate PR with that approach and we'll know if any issue arises. Till then, let's keep this PR open.
is that OK ?